src/components/TaskEntry.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+TaskEntry.propTypes = {
+  nameOfCrop: PropTypes.string,
+  date: PropTypes.string,
```

Comment:
is this a string or a date object? if date you should use `PropTypes.instanceOf(Date)`

as got to the end of the file i found out that it seems to be a string. maybe think about naming it `dateString` to make it more clear